test/wgpu/simple_test.exs

```diff
@@ -59,7 +59,7 @@ defmodule VideoCompositor.Wgpu.Test do
         "split[b], pad=iw:ih*2[src], [src][b]overlay=0:h"
       )
 
-      Utility.compare_contents(out_path, ref_path)
+      Utility.compare_contents_with_error(out_path, ref_path, 0.005)
```

Comment:
Since 0.005 is a value that we repeat (and probably will repeat in a future) mabe you should create a config and put it there?

Reply:
What do you think of making 0.005 a default value for this parameter?